app/src/main/java/com/github/swent/echo/compose/components/EventInfoSheet.kt

```diff
@@ -90,7 +90,7 @@ fun EventInfoSheet(
             )
             Text(
                 modifier = Modifier.padding(top = 24.dp),
-                text = event.organizerName,
+                text = event.organizer!!.name,
```

Comment:
I think this needs a safety check somewhere to not crash the app if organizer is null... the organizer should just not be showed (but this might be out of scope for this PR)

Reply:
Added this to a new issue: #98 .